### PR TITLE
Minor style and linting changes

### PR DESCRIPTION
**Pull request by sakshamsaxena**
GH maybe rendering tabs differently, so it may seem odd here. Looks fine in Brackets though. 
I don't really think this PR would make any visible impact in its working at any stage, but I think it was a good way to familiarize myself with the code.

### CODE REVIEW

File: javascripts/content_scripts/privly.js [hunk 1]
```diff
@@ -378,10 +372,11 @@ var privly = {
        };
 
     //Styling and display attributes
-     for(var key in attrs) 
-     {
-       iFrame.setAttribute(key, attrs[key]);
-     }
+    for(var key in attrs) {
+	    if (iFrame) {
+		    iFrame.setAttribute(key, attrs[key]);
```
Comment (smcgregor) [inline javascripts/content_scripts/privly.js]: Why are you testing for the existence of the iframe? It was created a few lines earlier.

File: javascripts/content_scripts/privly.js [hunk 2]
```diff
@@ -763,8 +754,6 @@ var privly = {
 
     "use strict";
 
-    return;
```
Comment (smcgregor) [inline javascripts/content_scripts/privly.js]: This should also not change.

**Comment by smcgregor** [comment]
It is a great way to familiarize yourself with the code! Aside from my comments, everything could be merged as-written. Feel free to update the pull request (instead of closing it).

**Comment by sakshamsaxena** [comment]
Haha. Merge what? Linting corrections? Thanks but no thanks. This PR isn't necessary or relevant. I'd rather take some more time so that I'm a regular user, and maybe then I can consider hacking into it again.

**Comment by smcgregor** [comment]
Sounds good.

**Pull request closed without merging**